examples/script_nifti.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pathlib
+import shutil
+import sys
+from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+import tempfile
+import urllib.request as ur
+from zipfile import ZipFile
+
+
+def main():
+    scriptPath = pathlib.Path(__file__).parent.absolute()  # Folder where this script is located
+    dataPath = os.path.join(scriptPath, 'data_testing')  # Folder where the data will be downloaded
+
+    # Download data when not already present
+    if not os.path.isdir(dataPath):
+        url = 'https://github.com/shimming-toolbox/data-testing/archive/r20200713.zip'
```

Comment:
With these sorts of hard-coded values, the convention is to write them as globals and in caps:

```
TEST_DATA='https://github.com/shimming-toolbox/data-testing/archive/r20200713.zip'
```

The other convention I've seen is to use `git submodules` to incorporate data by git address instead of writing your own downloader. This is what `datalad` advocates.